.taskcluster.yml

```diff
@@ -146,6 +146,38 @@ tasks:
           owner: ${user}@users.noreply.github.com
           source: ${repository}/raw/${head_rev}/.taskcluster.yml
 
+      - taskId: {$eval: as_slugid("docker_build")}
+        created: {$fromNow: ''}
+        deadline: {$fromNow: '1 hour'}
+        provisionerId: aws-provisioner-v1
+        workerType: releng-svc
+        payload:
+          capabilities:
+            privileged: true
+          maxRunTime: 3600
+          image: babadie/taskboot:latest
+          env:
+            GIT_REPOSITORY: ${repository}
+            GIT_REVISION: ${head_rev}
+          command:
+            - taskboot
+            - build
+            - infra/dockerfile.base
+            - --write
+            - /base.tar
+          artifacts:
+            public/bugbug-base.tar:
+              expires: {$fromNow: '2 weeks'}
```

Comment:
Maybe we could set a lower expiration, so that we don't upload a lot of data to S3? (if we have many PRs, with many iterations)